`opam install` & `opam update` fail silently if rsync is not available

In cases where remote repositories are fetched with `rsync`, if `rsync` is missing, `opam` doesn't provide useful information.

In `opam install` case, the error shown is something like

```
[ERROR] The sources of the following couldn't be obtained, aborting:
          - pkg-a.0.1
          - pkg-b.0.1
```

In `opam update` case:

```
<><> Synchronising development packages <><><><><><><><><><><><><><><><><><><><>
'opam update' failed.
```

It would be helpful if opam checked if `rsync` is available before trying to fetch the repositories,  and show an error that explains why the repositories can't be obtained in case it's missing, so users can understand better what's happening.

--- a/src/client/opamSolution.ml
+++ b/src/client/opamSolution.ml
@@ -190,8 +190,7 @@ let display_error (n, error) =
   | `Reinstall nv      -> f "recompiling" nv
   | `Remove nv         -> f "removing" nv
   | `Build nv          -> f "compiling" nv
-  | `Fetch nv          ->
-    if OpamConsole.verbose () then f "fetching sources for" nv
+  | `Fetch nv          -> f "fetching sources for" nv
 
 module Json = struct
   let output_request request user_action =

